features/admin/quota.feature

```diff
@@ -7,6 +7,7 @@ Feature: Quota related scenarios
   @gcp-ipi
   @4.9
   @aws-upi
+  @vsphere-ipi
   Scenario Outline: The quota usage should be incremented if meet the following requirement
```

Comment:
For `Scenario Outline` please add the tag to the exact example header. See examples in https://github.com/openshift/verification-tests/pull/2332/files

Reply:
It's fine there, as all the examples in this outline are able to run on all iaas.
But for scenario outlines which have examples on different iaas, it is a must to add tag to exact example header as suggest by @jhou1 stated above.